static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -27,31 +36,47 @@ const adSlotClassSelectorSizes = {
 	minBelow: 500,
 };
 
+const wrapSlotInContainer = (
+	ad: HTMLElement,
+	options: ContainerOptions = {},
+) => {
+	const container = document.createElement('div');
+	container.className = 'ad-slot-container ad-slot--offset-right';
+
+	if (options.sticky) {
+		ad.style.cssText += 'position: sticky; top: 10px;';
```

Comment:
Very minor: when they're sticky, the [right slot](https://github.com/guardian/dotcom-rendering/blob/a401bd0fbb839916af4fade22e8d1a3b2974b373/dotcom-rendering/src/web/components/TopRightAdSlot.importable.tsx#L71) and the [most viewed element](https://github.com/guardian/dotcom-rendering/blob/1372da38ea48e32c2157e815f698e2a86cb5760a/dotcom-rendering/src/web/components/MostViewedRight.tsx#L17) both have top 0px so we could consider doing the same here for consistency.

Reply:
This is an interesting one, based on a discussion we had 10px was suggested as a buffer from the top. But I appreciate it'd actually be good to keep consistency, so I'll change it to 0px and we can always change it/the others in the future.